packages/backend/src/graphql/queries/get-flows.ts

```diff
@@ -49,14 +56,20 @@ const getFlows: QueryResolvers['getFlows'] = async (
         .whereIn('flow_id', filteredFlowIds)
         .withSoftDeleted()
     })
-    .innerJoin('filtered_steps', 'id', 'filtered_steps.flow_id')
+    .innerJoin('filtered_steps', 'flows.id', 'filtered_steps.flow_id')
+    .whereIn('flows.id', filteredFlowIds)
+
+  flowsQuery
```

Comment:
why is this split up and duplicated .whereIn('flow.id', filteredFlowIds) shouldn't be required?